docs/Architectural-Overview.md

```diff
@@ -1,55 +1,87 @@
-
 ## Why moving from CTP and Avocado based test framework
-CTP OS and Avocado based test framework was used to develop automated test cases in Cortx R1/MVP. It had a binding to CentOS image which has Avocado and helper RPMs pre-installed for automation. Tests were written and run assuming prerequisites Python packages were installed. There was a hard dependency on CTP OS clients to run test automation. This framework was primarily aimed at testing firmware products and not for private cloud based solutions. It lacked parallel and distributed test execution capabilities at that point in time.
+
+CTP OS and Avocado based test framework was used to develop automated test cases in Cortx R1/MVP. It had a binding to
```

Comment:
The html preview or page adjusts the line length and does the wrap.  Is there a need to make this  into multiple lines?

Reply:
Looks like auto-formatting screwed up few things and fixed many things :( fixing by hand now.